File: modul3/Python/PRAK304-2310817210029-Putra_Whyra_Pratama_Setiawan.py
a = int(input(""))
if a == 0:
    print("Nol")
elif a > 0 and a < 10:
    print("Satuan")
elif a == 10:
    print("Puluhan")
elif a > 10 and a < 20:
    print("Belasan")
elif a > 19 and a < 100:
    print("Puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")

File: modul3/C/PRAK304-2310817210029-Putra_Whyra_Pratama_Setiawan.c
#include <stdio.h>
int main() {
    int a;
    scanf("%d", &a);
    if(a == 0) {
        printf("Nol");
    } else if(a > 0 && a < 10) {
        printf("Satuan");
    } else if(a == 10) {
        printf("Puluhan");
    } else if(a > 10 && a < 20) {
        printf("Belasan");
    } else if(a > 19 && a < 100) {
        printf("Puluhan");
    } else {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
}